app/assets/javascripts/application.js

```diff
@@ -883,9 +881,587 @@ function fb_login() {
+            alt: 'Add Comment Profile Picture',
+            class: 'addCommentPic',
+            height: 32,
+            width: 32
+          }));
+
+          // Add Comment Input Field
+          $('#comments' + post.id).append($('<input>', {
+            type: 'text',
+            class: 'addCommentField',
+            id: 'addCommentField' + post.id,
+            name: 'addComment' + post.id,
+            placeholder: 'Write a comment...'
+          }));
+        }
+      }
+
+      // Click to Like
+      $('#wall a.notLiked').click(function() {
+        FB.api(
+          "/" + $(this).attr('id') + "/likes",
+          "POST",
+          function (response) {
+            if (response && !response.error) {
+              console.log('liked!');
+            } else {
+              console.log('error');
+            }
+          }
+        );
+        $(this).attr('class', 'likePost liked');
+        location.reload();
+      });
+
+      // Click to Unlike
+      $('#wall a.liked').click(function() {
+        FB.api(
+          "/" + $(this).attr('id') + "/likes",
+          "DELETE",
+          function (response) {
+            if (response && !response.error) {
+              console.log('unliked');
+            } else {
+              console.log('error');
+            }
+          }
+        );
+        $(this).attr('class', 'likePost notLiked');
+        location.reload();
+      });
+
+      // Click to Add Comment
+      $('#wall a.commentPost').click(function() {
+        $('#addCommentField' + $(this).attr('id').substring('commentPost'.length)).focus();
+      });
+
+      // Submit Post Comment
+      $('.addCommentField').keypress(function(event) {
+        if (event.keyCode == 13) { // Enter key
+          FB.api(
+            "/" + $(this).attr('id').substring('addCommentField'.length) + "/comments",
+            "POST",
+            {"message":$(this).val()},
+            function (response) {
+              if (response && !response.error) {
+                // Added comment
+              } else {
+                console.log('error');
+              }
+            }
+          );
+          location.reload();
+        }
+      });
+  });
+}   
+
+function updateStatus(status) {
+    FB.api('/me/feed',
+        'post',
+        {
+            message: status,
+            privacy: {
+                "value": $("#status_privacy").val()
+            }
+        });
+}
+
+function uploadPhoto(status, url) {
+    FB.api("/me/photos",
+        "POST",
+        {
+            "caption": status,
+            "url": url,
+            "privacy": {
+                "value": $("#status_privacy").val()
+            }
+        });
+}
+
+function logout() {
+    FB.logout();
+}
+
 /*********************************************************/
 /* CALENDAR */
 /*********************************************************/
+function renderFrontPageCalendar() {
+  $('#frontPageCalendar').fullCalendar({
+    defaultView: 'basicDay',
```

Comment:
Changed from agendaDay to basicDay 
List of events instead of calendar/agenda view